staging/src/k8s.io/apiserver/pkg/storage/cacher/metrics.go

```diff
@@ -56,12 +56,22 @@ var (
 		},
 		[]string{"resource"},
 	)
+
+	watchCacheCapacity = metrics.NewGaugeVec(
+		&metrics.GaugeOpts{
+			Name:           "watch_cache_capacity",
+			Help:           "number of watch cache capacity broken by resource type.",
```

Comment:
Total capacity of watch cache broken by resource type.

Reply:
ok, i have fixed this